<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>346</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>694</width>
    <height>346</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>694</width>
    <height>346</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Lista de Cadastrados</string>
  </property>
  <widget class="QListWidget" name="list_nome">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>251</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="list_idade">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>81</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MV Boli&quot;;
color: rgb(85, 0, 255);</string>
   </property>
   <property name="text">
    <string>Nome</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MV Boli&quot;;
color: rgb(85, 0, 255);</string>
   </property>
   <property name="text">
    <string>Idade</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MV Boli&quot;;
color: rgb(85, 0, 255);</string>
   </property>
   <property name="text">
    <string>GÃªnero</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="list_genero">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>60</y>
     <width>161</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>30</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MV Boli&quot;;
color: rgb(85, 0, 255);</string>
   </property>
   <property name="text">
    <string>Turno</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="list_turno">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>60</y>
     <width>161</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
